apps/dashboard/tests/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is the extra tsconfig.json required?

Reply:
yes it is for the tests to treat them as `commonjs` module, otherwise the `@novu/testing` package won't work